package.json

```diff
@@ -51,8 +51,9 @@
     "build": "babel --ignore=__tests__ src --out-dir lib",
     "lint": "standard",
     "prepublish": "npm test",
-    "posttest": "NODE_ENV=production npm run build",
+    "pretest": "npm run build",
```

Comment:
No, I don't think this is accurate. I could be wrong. But would you add a quick test to verify. Just a simple pass a number where a string is required. Since flow is runtime, we want it to be runtime for the tests, but in production, we don't need the runtime anymore. A quick test will tell you if I'm correct or not.